dist/obs-server.spec

```diff
@@ -341,7 +340,7 @@ export DESTDIR=$RPM_BUILD_ROOT
 
 export OBS_VERSION="%{version}"
 DESTDIR=%{buildroot} make install FILLUPDIR=%{_fillupdir}
-
+sed -n -e '/commit: /,$p' %{_sourcedir}/open-build-service.obsinfo | tr -d "commit: " > %{buildroot}/srv/www/obs/api/last_deploy
```

Comment:
@eduardoj maybe you have a more elegant way to achieve this:
I have obs-server.obsinfo file with the following content:

```
name: open-build-service
version: 2.10~alpha.20180507T142315.30c3301a1
mtime: 1525695795
commit: 30c3301a1d25b8bad1505a5e16ff287289934b6c
```

and I want to write the commit hash into the ``last_deploy`` file.

Reply:
`awk -F: 'END{print $2}' file`

Reply:
@ChrisBr, this works for me:

`sed -n -e 's/commit: \(.\+\)/\1/p' %{_sourcedir}/open-build-service.obsinfo > %{buildroot}/srv/www/obs/api/last_deploy`